<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="defaultElapsed">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="BenchmarkView" name="defaultView"/>
     </item>
     <item row="1" column="2">
      <widget class="BenchmarkViewUpdateEach" name="updateCompressionView"/>
     </item>
     <item row="1" column="1">
      <widget class="BenchmarkViewJoinRects" name="rectJoinView"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Manual rect join</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Default implementation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Update event compression</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="rectJoinElapsed">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="updateCompressionElapsed">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="update4ItemsButton">
       <property name="text">
        <string>Update 4 items</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateAllButton">
       <property name="text">
        <string>Update all</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BenchmarkView</class>
   <extends>QTableView</extends>
   <header>benchmarkview.h</header>
  </customwidget>
  <customwidget>
   <class>BenchmarkViewJoinRects</class>
   <extends>QTableView</extends>
   <header>benchmarkview.h</header>
  </customwidget>
  <customwidget>
   <class>BenchmarkViewUpdateEach</class>
   <extends>QTableView</extends>
   <header>benchmarkview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
